src/index.html

```diff
@@ -22,6 +22,362 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <div class="catalog">
+      <header class="catalog__header">
+        <a
+          class="catalog__header-logo"
+          href="/"
+        >
+          <img
+            src="./images/logo.png"
+            alt="MOYO logo"
+          />
+        </a>
+        <nav>
+          <ul class="catalog__header-navigation">
+            <li>
+              <a
+                class="catalog__header-navigation-list is-active"
+                href="/"
+              >
+                Apple
+              </a>
+            </li>
+            <li>
+              <a
+                class="catalog__header-navigation-list"
+                href="/"
+              >
+                Samsung
+              </a>
+            </li>
+            <li>
+              <a
+                class="catalog__header-navigation-list"
+                href="/"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li>
+              <a
+                data-qa="nav-hover"
+                class="catalog__header-navigation-list"
+                href="/"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li>
+              <a
+                class="catalog__header-navigation-list"
+                href="/"
+              >
+                Gadgets
+              </a>
+            </li>
+            <li>
+              <a
+                class="catalog__header-navigation-list"
+                href="/"
+              >
+                Tablets
+              </a>
+            </li>
+            <li>
+              <a
+                class="catalog__header-navigation-list"
+                href="/"
+              >
+                Photo
+              </a>
+            </li>
+            <li>
+              <a
+                class="catalog__header-navigation-list"
+                href="/"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+
+      <main class="catalog__main">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card-picture"
+            src="./images/imac.jpeg"
+            alt="Product picture"
+          />
+          <h1 class="card-description">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+          <h2 class="card-code">Product code: 195434</h2>
+          <div class="card-reviews">
+            <div class="card-reviews-stars">
+              <div class="reviews-stars-star"></div>
+              <div class="reviews-stars-star"></div>
+              <div class="reviews-stars-star"></div>
+              <div class="reviews-stars-star"></div>
+              <div class="reviews-stars-star"></div>
+            </div>
```

Comment:
```suggestion
            <div class="card-reviews-stars stars stars--4">
              <div class="reviews-stars-star stars__star"></div>
              <div class="reviews-stars-star stars__star"></div>
              <div class="reviews-stars-star stars__star"></div>
              <div class="reviews-stars-star stars__star"></div>
              <div class="reviews-stars-star stars__star"></div>
            </div>
```
Consider using the structure from the previous task